### PR TITLE
Create .about.yml

### PR DESCRIPTION
**Pull request by gbinal**

### CODE REVIEW

**Comment by gbinal** [comment]
moving from https://github.com/18F/data-private/blob/master/projects/dap.yml

File: .about.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+project: "Analytics.USA.gov"
+name: dap
+github:
+- gsa/analytics.usa.gov
```
Comment (konklone) [inline .about.yml]: We're now using `18f/analytics.usa.gov`. I'll change on-branch.

File: .about.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- 18f/analytics-reporter
+description: A lightweight dashboard that shows web traffic across the .gov domain.  
+partners:
+- General Services Administration
+stage: beta
```
Comment (konklone) [inline .about.yml]: Are we still in `beta`? I'd argue we're just a regular product.

File: .about.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+milestones:
+- 'May 2012: Digital Government Strategy calls for a government-wide analytics program'
+- 'May 2013: Digital Analytics Program announced'
+- 'August 2014: Full Digital Analytics Program access expanded to all federal staff at participating agencies'
+- 'March 2015: Analytics.USA.gov announced'
+contact:
+- dap@gsa.gov
+stack: jekyll, bourbon, neat, d3, node, analytics-reporter
+team: gray, eric, leah, shawn
```
Comment (konklone) [inline .about.yml]: The team has changed since then, and now includes Gabriel and Julia and potentially others.

File: .about.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+licenses:
+  analytics.usa.gov: Public Domain (CC0)
+links:
+- https://analytics.usa.gov
+- https://github.com/gsa/analytics.usa.gov
```
Comment (konklone) [inline .about.yml]: I'll switch this link to `18f` and add analytics-reporter.

File: .about.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+partners:
+- General Services Administration
+stage: beta
+milestones:
+- 'May 2012: Digital Government Strategy calls for a government-wide analytics program'
+- 'May 2013: Digital Analytics Program announced'
+- 'August 2014: Full Digital Analytics Program access expanded to all federal staff at participating agencies'
+- 'March 2015: Analytics.USA.gov announced'
+contact:
+- dap@gsa.gov
```
Comment (konklone) [inline .about.yml]: @tdlowden I think this email changed to `dap@listserv.gsa.gov`?

**Comment by onceuponatimeforever** [comment]
What is the screenshot of this new page?

**Comment by gbinal** [comment]
Hey - @onceuponatimeforever!   This yml file won't have an html presence just for this project.  Rather, we're working to scale out this schema in repos for projects, guilds, etc so that we can power things like our [Team API](https://18f.gsa.gov/hub/api/), [Dashboard](https://18f.gsa.gov/dashboard), etc.  You can see more info here - https://github.com/18F/about_yml.

**Comment by gbinal** [comment]
Also, I'm going to go ahead and merge this as is, and we can make edits to it going forward.

**Pull request merged**